src/backend/aspen/workflows/pangolin/find_samples.py

```diff
@@ -16,19 +17,20 @@
 
 
 def check_latest_pangolin_version() -> str:
-    contents = urllib.request.urlopen(
-        "https://github.com/cov-lineages/pangoLEARN/releases/latest"
+    installed_version = (
+        subprocess.check_output(["pangolin", "-pv"]).decode("utf-8").strip()
     )
-    # get latest version from redirected url:
-    redirected_url: str = contents.url
-    latest_version: str = redirected_url.split("/")[-1]
-    return latest_version
+    installed_version = re.sub(".*?([0-9\\.]+)$", "\\1", installed_version)
+    return installed_version
 
 
 def should_sample_be_updated(sample: Sample, most_recent_pango_version: str) -> bool:
     if not sample.lineages:
         return True
-    return sample.lineages[0].lineage_software_version != most_recent_pango_version
+    sample_lineage_version = re.sub(
```

Comment:
I'd like either a comment on this var declaration or a docstring for the `should_sample_be_updated` function: in the context of this PR with the description, it makes sense what you're trying to do -- although I'll be honest, it took me quite a bit of looking at that regex to make sense of what it was doing, but I'm not the best at regex -- but I think without the PR description as context, why this is necessary would be pretty mystifying.